### DISCUSSION TITLE
list autoscan directories

### DISCUSSION THREAD

**Discussion by jtmoon79** [gerbera/gerbera#2676] [Q&A] [answered]
How do I list Autoscan directories ?

From the gerber web interface, I have added several Filesystem paths using the button ⊕ 
 with overlay text _add as autoscan dir_ .  How can I review the Filesystem paths add this way?

I presume _autoscan dir_ means a path is saved for repeated scanning on some schedule. Thus those paths would need to be recorded somewhere, and could be listed.

**Comment by KarlStraussberger** [comment]
All your changes on web ui are stored in gerbera database (either sqlite or mysql). So you need some SQL knowledge to extract info from `mt_autoscan` table.

**Comment by jtmoon79** [comment, marked as answer]
This shell command worked

```text
$ sqlite3 /var/lib/gerbera/gerbera.db \
  -cmd '.headers on' \
  -cmd '.mode column' \
  'SELECT mt_autoscan.id, mt_autoscan.scan_level, mt_cds_object.location, mt_autoscan.recursive, mt_autoscan.hidden, mt_autoscan.interval
  FROM mt_autoscan
  JOIN mt_cds_object
  ON mt_autoscan.obj_id=mt_cds_object.id;
  '
```

output table like
```text
id  scan_level  location          recursive  hidden  interval
--  ----------  ----------------  ---------  ------  --------
2   full        D/srv/books       1          0       86000
3   full        D/srv/movies      1          0       86000
4   basic       D/srv/music       0          0       86400
```
